modules/ROOT/pages/clustering/setup/discovery.adoc

```diff
@@ -4,66 +4,167 @@
+* xref:configuration/configuration-settings.adoc#config_dbms.kubernetes.discovery.v2.service_port_name[`dbms.kubernetes.discovery.v2.service_port_name`] to specify the name of the service port name used in the Kubernetes API when using version v2.
+
+The following sections describe the different methods for server discovery, configuration settings, and examples of how to move from discovery service v1 to v2.
+
+[[clustering-discovery-methods]]
+== Methods for server discovery
+
+Depending on the type of xref:configuration/configuration-settings.adoc#config_dbms.cluster.discovery.resolver_type[`dbms.cluster.discovery.resolver_type`] currently in use, the discovery service can use a list of server addresses, DNS records, or Kubernetes services to discover other servers in the cluster.
+The discovery configuration is used for initial discovery and to continuously exchange information about changes to the topology of the cluster.
+
+[[clustering-discovery-list]]
+=== Discovery using a list of server addresses
 
 If the addresses of the other cluster members are known upfront, they can be listed explicitly.
-This is convenient, but has limitations:
+However, this method has limitations, such as:
 
 * If servers are replaced and the new members have different addresses, the list becomes outdated.
 An outdated list can be avoided by ensuring that the new members can be reached via the same address as the old members, but this is not always practical.
 * Under some circumstances the addresses are unknown when configuring the cluster.
 This can be the case, for example, when using container orchestration to deploy a cluster.
 
-Additional mechanisms for using DNS are provided for the cases where it is not practical or possible to explicitly list the addresses of cluster members to discover.
+To use this method, set `dbms.cluster.discovery.resolver_type=LIST` and hard code the addresses in the configuration of each server.
+For example:
 
-The discovery configuration is just used for initial discovery and a running cluster continuously exchanges information about changes to the topology.
-The behavior of the initial discovery is determined by the parameters `xref:configuration/configuration-settings.adoc#config_dbms.cluster.discovery.resolver_type[dbms.cluster.discovery.resolver_type]` and `xref:configuration/configuration-settings.adoc#config_dbms.cluster.discovery.endpoints[dbms.cluster.discovery.endpoints]`, and is described in the following sections.
+[.tabbed-example]
+=====
+[role=include-with-discovery-service-v1 label--deprecated-5.23]
+======
+[source, parameters]
+----
+dbms.cluster.discovery.resolver_type=LIST
 
+server.discovery.advertised_address=server01.example.com:5000
+dbms.cluster.discovery.endpoints=server01.example.com:5000,server02.example.com:5000,server03.example.com:5000
+dbms.cluster.discovery.version=V1_ONLY
+----
+======
+[role=include-with-discovery-service-v2 label--new-5.23]
```

Comment:
I think we have this as `label--new-5.22` to keep consistent with the important notification above which says that v2 was introduced in 5.22